src/languages/nl.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    TITLE_OFFLINE_WARNING: "Gebruik een offline computer",
+    CONTENT_OFFLINE_WARNING: "Mits je dit nog niet hebt gedaan, download deze pagina en gebruik hem op een offline computer om je persoonlijke sleutels veilig te houden!",
+
+    LABEL_APP_LANGUAGE: "Taal",
+
+    TITLE_RECOVER_WALLET: "Vul 'mnemonic' in om de NEM accounts te herstellen",
+    LABEL_NETWORK: "Netwerk",
+    LABEL_BIP39_MNEMONIC: "BIP-39 'mnemonic'",
```

Comment:
People seem to use the word "ezelsbruggetje" to refer to BIP-39 mnemonics. Is there any reason not to use it here?